cranelift-codegen/meta/src/cdsl/types.rs

```diff
@@ -31,6 +32,7 @@ static _RUST_NAME_PREFIX: &'static str = "ir::types::";
 pub enum ValueType {
     BV(BVType),
     Lane(LaneType),
+    Reference(ReferenceType),
```

Comment:
Maybe put this in `LaneType` instead.

Reply:
We decided not to put reference types as a LaneType because LaneTypes are vectorizable while references are not. We need a logical separation for parts such as gen_types.rs:33 which generates vectors by iterating through all LaneTypes.